backend/Dockerfile

```diff
@@ -1,14 +1,32 @@
-FROM golang:1.10.2-stretch
+FROM golang:1.10.2
 ENV SCOPE_SKIP_UI_ASSETS true
-RUN apt-get update && \
+RUN set -eux; \
+   export arch_val="$(dpkg --print-architecture)"; \
+   if [ "$arch_val" != "ppc64el" ]; then \
+     apt-get update && \
 	apt-get install -y libpcap-dev python-requests time file shellcheck git gcc-arm-linux-gnueabihf curl build-essential python-pip && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
+   else \
+     apt-get update && \
+	apt-get install -y libpcap-dev python-requests time file shellcheck git curl build-essential python-pip && \
+	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
+   fi;
+
 RUN go clean -i net && \
 	go install -tags netgo std && \
-	go install -race -tags netgo std
-RUN curl -fsSL -o shfmt https://github.com/mvdan/sh/releases/download/v1.3.0/shfmt_v1.3.0_linux_amd64 && \
-    chmod +x shfmt && \
-    mv shfmt /usr/bin
+   export arch_val="$(dpkg --print-architecture)"; \
+   if [ "$arch_val" != "ppc64el" ]; then \
+	go install -race -tags netgo std; \
+   fi
+RUN export arch_val="$(dpkg --print-architecture)"; \
+   if [ "$arch_val" != "ppc64el" ]; then \
```

Comment:
surely if the thing we are trying to download is `amd64` this test should be `== amd64` ?